linux/xdg/src/main/java/com/github/weisj/darkmode/platform/linux/xdg/FreedesktopInterface.kt

```diff
@@ -0,0 +1,17 @@
+package com.github.weisj.darkmode.platform.linux.xdg
+
+import org.freedesktop.dbus.annotations.DBusInterfaceName
+import org.freedesktop.dbus.interfaces.DBusInterface
+import org.freedesktop.dbus.messages.DBusSignal
+import org.freedesktop.dbus.types.Variant
+
+@DBusInterfaceName("org.freedesktop.portal.Settings")
+interface FreedesktopInterface : DBusInterface {
+    fun Read(namespace: String, key: String): Variant<*>
+
+    class SettingChanged(objectpath: String, namespace: String, key: String, value: Variant<Any>) :
+        DBusSignal(objectpath, namespace, key, value) {
+        val colorSchemeChanged: Boolean =
+            namespace == "org.freedesktop.appearance" && key == "color-scheme"
```

Comment:
Those constants are repeated in `XdgThemeMonitorService`. Please make them constants in `FreeDesktopInterface`
````kotlin
object {
    const val APPEARANCE_NAMESPACE = "org.freedesktop.appearance"
    const val COLOR_SHEME_KEY = "color-scheme"
}
````